rke/templates/templates.go

```diff
@@ -173,9 +174,13 @@ func LoadK8sVersionedTemplates() map[string]map[string]string {
 			">=1.21.0-rancher0 <1.21.14-rancher1-1": aciv500,
 			">=1.21.14-rancher1-1 <1.22.0-rancher0": aciv523,
 			">=1.22.0-rancher0 <1.22.11-rancher1-1": aciv500,
-			">=1.22.11-rancher1-1 <1.23.0-rancher0": aciv523,
+			">=1.22.11-rancher1-1 <1.22.13-rancher1-1": aciv523,
+			">=1.22.13-rancher1-1 <1.23.0-rancher0": aciv5233,
 			">=1.23.0-rancher0 <1.23.8-rancher1-1":  aciv500,
-			">=1.23.8-rancher1-1":                   aciv523,
+			">=1.23.8-rancher1-1 <1.23.10-rancher1-1":  aciv523,
+			">=1.23.10-rancher1-1 <1.24.0-rancher0":  aciv5233,
+			">=1.24.0-rancher0 <1.24.2-rancher1-1":  aciv523,
+			">=1.24.2-rancher1-1":                   aciv5233,
```

Comment:
Same as above, `v1.24.4-rancher1-2` is the first version where `aciv5233` should apply to.

Reply:
addressed